src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -2581,6 +2581,22 @@ public function setComments(array $comments)
 
         return $this;
     }
+    
+
+    /**
+     * Remove comment.
+     *
+     * @param int $columnIndex Numeric column coordinate of the cell
+     * @param int $row Numeric row coordinate of the cell
+     *
+     */
+    public function removeCommentByColumnAndRow($columnIndex, $row)
```

Comment:
Useful to have a method to delete a cell comment, although you may have noticed that I'm trying to deprecate "byColumnAndRow" methods in favour of the cellAddress methods.
Every existing functionality has a cellAddress (or cellRange) method, not ever functionality has a "byColumnAndRow" method, so we already have inconsistency; and providing only a "byColumnAndRow" method for this functionality introduces even more inconsistency.
The `master` branch also has a slightly different approach for validating the address, which allows passing in a string with the cell address, an array of column/row ids, or a CellAddress object

So you might want to consider using the cellAddress approach instead (or at last as well as the "byColumnAndRow" method)